src/main/java/org/sonarlint/intellij/core/ProjectBindingManager.java

```diff
@@ -69,7 +71,7 @@ public synchronized ConnectedSonarLintEngine getConnectedEngineSkipChecks() {
     return engineManager.getConnectedEngine(projectSettings.getServerId());
   }
 
-  public synchronized ConnectedSonarLintEngine getConnectedEngine() {
+  public synchronized ConnectedSonarLintEngine getConnectedEngine() throws InvalidBindingException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.sonarlint.intellij.exception.InvalidBindingException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)